client/flutter/lib/generated/intl/messages_en.dart

```diff
@@ -55,7 +55,7 @@ class MessageLookup extends MessageLookupByLibrary {
     "newsFeedSliverListNewsFeedItemDescription1" : MessageLookupByLibrary.simpleMessage("Situation reports provide the latest updates on the novel coronavirus outbreak."),
     "newsFeedSliverListNewsFeedItemDescription2" : MessageLookupByLibrary.simpleMessage("Rolling updates on coronavirus disease (COVID-19) sourced from across WHO media."),
     "newsFeedSliverListNewsFeedItemDescription3" : MessageLookupByLibrary.simpleMessage("All news releases, statements, and notes for the media."),
-    "newsFeedSliverListNewsFeedItemDescription4" : MessageLookupByLibrary.simpleMessage("Coronavirus disease (COVID019) press briefings including videos, audio and transcripts."),
+    "newsFeedSliverListNewsFeedItemDescription4" : MessageLookupByLibrary.simpleMessage("Coronavirus disease (COVID-2019) press briefings including videos, audio and transcripts."),
```

Comment:
```suggestion
    "newsFeedSliverListNewsFeedItemDescription4" : MessageLookupByLibrary.simpleMessage("Coronavirus disease (COVID-19) press briefings including videos, audio and transcripts."),
```

Reply:
heads up @advayDev1 that these files are all generated by `flutter_intl`, meaning that without updating the source in the `.arb` files, the next time we run the generator, these changes will be overwritten... might be worth a separate PR to update the originals